widget/edit.html

```diff
@@ -15,31 +15,27 @@ <h3 class="modal-title col-md-9">Edit Funnel Chart</h3> <button type="button" cl
         </div>
         <div
             data-ng-class="{ 'has-error': editFunnelWidgetForm.funnelModuleType.$invalid && editFunnelWidgetForm.funnelModuleType.$touched }">
-            <label for="funnelModuleType" class="control-label">Select Dataset</label><span class="text-danger">*
+            <label for="funnelModuleType" class="control-label">Select Data Source</label><span class="text-danger">*
                 <span
-                    data-uib-tooltip="Select the 'Single Module' option if all the records to be rendered are in a single record of a certain field of the module.
-                'Across Modules' Dataset lets you select the module and it displays the count of records for the module based on the filters given"
+                    data-uib-tooltip="Select the 'Single Record' option if all the records to be rendered are in a single record of a certain field of the module.
+                'Multiple Modules' Dataset lets you select the module and it displays the count of records for the module based on the filters given"
```

Comment:
dataset or datasource?
Multiple module or across modules?

Reply:
Multiple modules

Reply:
Data source and multiple modules is in the final draft